memdown.d.ts

```diff
@@ -1,55 +1,25 @@
-import * as Abstract from 'abstract-leveldown';
+import { AbstractLevelDOWN } from 'abstract-leveldown';
 
-declare namespace memdown {
-  export interface MemDown<TKey, TValue>
-    extends Abstract.LevelDOWN<
-    TKey,
-    TValue,
-    MemDownOptions,
-    MemDownPutOptions,
-    MemDownGetOptions,
-    MemDownDeleteOptions,
-    MemDownIteratorOptions<TKey, TValue>,
-    MemDownBatchOptions> {
-  }
-
-  export interface MemDownOptions {
-  }
-
-  export interface MemDownPutOptions {
-  }
-
-  export interface MemDownGetOptions {
-    asBuffer?: boolean;
-  }
-
-  export interface MemDownDeleteOptions {
-  }
+export interface MemDown<K=any, V=any>
+  extends AbstractLevelDOWN<K, V, {}, {}, MemDownGetOptions, {}, MemDownIteratorOptions, {}> {
+}
 
-  export interface MemDownIteratorOptions<K, V> {
-    gt?: K;
-    gte?: K;
-    lt?: K;
-    lte?: K;
-    reverse?: boolean;
-    keys?: boolean;
-    values?: boolean;
-    limit?: number;
-    keyAsBuffer?: boolean;
-    valueAsBuffer?: boolean;
-  }
+interface MemDownConstructor {
+  new <K=any, V=any>(location: string): MemDown<K, V>;
+  <K=any, V=any>(location: string): MemDown<K, V>;
+}
 
-  export interface MemDownBatchOptions {
-  }
+export interface MemDownGetOptions {
+  asBuffer?: boolean;
+}
 
-  function clearGlobalStore(strict?: boolean);
-  function destroy(location: string, cb: () => void): void;
+export interface MemDownIteratorOptions {
+  keyAsBuffer?: boolean;
+  valueAsBuffer?: boolean;
 }
 
-declare function memdown<
-  TKey=any,
-  TValue=any
-  >(location: string)
-  : memdown.MemDown<TKey, TValue>;
+export function clearGlobalStore(strict?: boolean);
+export function destroy(location: string, cb: () => void): void;
 
-export = memdown;
\ No newline at end of file
+declare const MemDown: MemDownConstructor;
+export default MemDown;
```

Comment:
I'd like to suggest to also add `export { MemDown }` at the end of the file.

This will enable the user to use either `import { MemDown } from 'memdown'` or `import MemDown from 'memdown'`